testnet-contracts/contracts/Bank.sol

```diff
@@ -0,0 +1,96 @@
+pragma solidity ^0.5.0;
+
+import "../../node_modules/openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "./BankToken.sol";
+
+/**
+ * @title Bank
+ * @dev Manages the deployment and minting of ERC20 compatible BankTokens
+ **/
+
+contract Bank {
+
+    using SafeMath for uint256;
+
+    mapping(address => bool) public bankTokens;
+    uint256 public numbTokens;
```

Comment:
numbTokens...i read that as numb tokens. Maybe rather call it numTokens?

Reply:
Updated to ```cosmosTokenCount``` in https://github.com/cosmos/peggy/pull/76/commits/89cb10e8d06ff3dcd467bcfc7b2821bf14240153